recon_surf/compat_long_outputs.py

```diff
@@ -0,0 +1,338 @@
+                raise FastSurferCompatError(
+                    f"You are trying to run recon-surf with FreeSurfer version {version_content}. We are currently "
+                    f"supporting only FreeSurfer {fs_support}. Therefore, make sure to export and source the correct "
+                    f"FreeSurfer version before running this script:\n"
+                    f"  export FREESURFER_HOME=/path/to/your/local/fs{fs_support}\n"
+                    f"  source $FREESURFER_HOME/SetUpFreeSurfer.sh"
+                )
+        else:
+            FastSurferCompatError("Could not find/read FreeSurfer build-stamp file.")
+
+def softlink_or_copy(source: str | Path, target: str | Path) -> None:
+    """Create soft link or copy file if linking fails"""
+    source_path = Path(source)
+    target_path = Path(target)
+
+    if target_path.exists():
+        target_path.unlink()
+
+    try:
+        target_path.symlink_to(source_path)
+    except OSError:
+        # If symlink fails, copy the file
+        import shutil
+
+        if not target_path.is_absolute():
+            target_path = (source.parent / target_path).resolve()
+        shutil.copy2(source_path, target_path)
+
+def get_voxel_size(image_file: Path) -> float:
+    """Get voxel size from aseg file using nibabel."""
+    try:
+        img = nib.load(str(image_file))
+        return float(img.header.get_zooms()[0])
+    except Exception as e:
+        raise FastSurferCompatError(f"Could not read voxel size from {image_file}: {e}") from e
+
+@lru_cache
+def get_supported_freesurfer_version() -> str:
+    """Get FreeSurfer version from FreeSurfer from recon-surf.sh."""
+    recon_surf_script = Path(__file__).parent / "recon-surf.sh"
+    with recon_surf_script.open() as fp:
+        for line in fp.readlines():
+            if line.lstrip().startswith("FS_VERSION_SUPPORT"):
+                return line.strip().removeprefix("FS_VERSION_SUPPORT").lstrip(" =").strip("\"")
+    return "7.4.1"
+
+def make_parser() -> argparse.ArgumentParser:
+    """Parse command line arguments"""
+    from FastSurferCNN.utils.arg_types import int_gt_zero
+
+    parser = argparse.ArgumentParser(
+        description="compat_long_outputs.py takes a longitudinally processed subject and creates files "
+                    "missing for other longitudinal processing like the hippocampal subfields stream of FreeSurfer.",
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        epilog="""
+               REFERENCES:
+               
+               If you use this for research publications, please cite:
+               
+               Henschel L, Conjeti S, Estrada S, Diers K, Fischl B, Reuter M, FastSurfer - A
+                fast and accurate deep learning based neuroimaging pipeline, NeuroImage 219
+                (2020), 117012. https://doi.org/10.1016/j.neuroimage.2020.117012
+                
+               Henschel L*, Kuegler D*, Reuter M. (*co-first). FastSurferVINN: Building
```

Comment:
```suggestion
               Henschel L, Kuegler D, Reuter M. FastSurferVINN: Building
```
avoid asterix as it is interpreted as emphasis.